src/components/modebar/manage.js

```diff
@@ -194,7 +193,7 @@ function isSelectable(fullData) {
 
         var trace = fullData[i];
 
-        if(!trace._module || !trace._module.selectPoints) continue;
+        if(!trace._module || !trace._module.getPointsIn || !trace._module.toggleSelected) continue;
```

Comment:
It might be better to make `Plotly.register` set a boolean `_module.selected` somewhere [here](https://github.com/plotly/plotly.js/blob/24b291285a15650a76ae6c78e2762a8e88f2027d/src/registry.js#L224-L267) and use that boolean downstream (e.g. in `Plots.supplyDefaults`) instead of looking for `getPointsIn` and `toggleSelected`.

Reply:
Neat! Would `_module.selectable` even be a better name?